trackpy/tests/test_feature.py

```diff
@@ -512,7 +512,7 @@ def test_minmass_maxsize(self):
         image = np.zeros(dims, dtype=np.uint8)
         pos1 = np.array([15, 20])
         pos2 = np.array([40, 40])
-        pos3 = np.array([25, 45])
+        pos3 = np.array([25, 50])
```

Comment:
This had to do with the removal of `filter_before`. Without `filter_before` the `maxsize` tests did not pass; probably because the locations of the features change slightly in refinement and therefore the `maxsize` threshold was met by different particles.

This had mainly to do with the fact that features were too close together. Therefore I increased the spacing so that the tests passed again.